hw2/04-background.html

```diff
@@ -6,23 +6,32 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
-  <body>
-    <main>
+  <body
+    class="container d-flex justify-content-center p-3"
+    style="background-color: #175f93"
+  >
+    <main class="bg-light rounded w-75 p-4 m-5">
       <h1>Exercise 04 - Background</h1>
       <p>
         The background color will change every few seconds (as indicated below).
         You can stop this behavior and you can also change how often the
         background color changes.
       </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
+      <div class="row">
+        <label for="userinput">Background change interval (seconds): </label>
```

Comment:
A more descriptive name for inputs may be helpful if we had a more complicated form with more inputs that could get confusing, something like 'interval seconds' rather than 'userinput' , but should be totally fine for a single input like this form. 

Also, it looks like there's a default value being set for the interval, which can be displayed at the default value displayed to prevent any confusion about the default value.

Reply:
You're right, I rushed into naming the IDs. and good idea, I added the value to show that background is changing every 3 seconds.